src/OpenTelemetry.Abstractions/Trace/AttributeValue.cs

```diff
@@ -72,6 +75,34 @@ public static IAttributeValue<double> DoubleAttributeValue(double doubleValue)
             return new AttributeValue<double>(doubleValue);
         }
 
+        public new abstract Type GetType();
```

Comment:
I would really avoid using `new` here, especially for such an elementary method as `GetType`. This will most likely lead to confusion. 
Having e.g. an `AttributeType` property, is much clearer.